_includes/manuals/1.23/1.2_release_notes.md

```diff
@@ -24,6 +24,45 @@ The UI team has been hard at work behind the scenes at making our new React-base
 #### Experimental Redis cache support
 Redis has been added as an experimental caching backend for Foreman instead of the default file-based store, or the memcache-based store which requires an additional plugin. It is still missing installer support and documentation - contributions are most welcome! ([#26918](https://projects.theforeman.org/issues/26918))
 
+### Release notes for 1.23.2
+#### Authentication
+* Wrong Error Message/Return code getting after exceeding the 'failed_login_attempts_limit' ([#28860](https://projects.theforeman.org/issues/28860))
+
+#### Compute resources
+* Custom field on host view ([#29059](https://projects.theforeman.org/issues/29059))
+
+#### Compute resources - oVirt
+* NoMethodError: undefined method \`storage_domain' for nil:NilClass ([#26919](https://projects.theforeman.org/issues/26919))
+* oVirt API v4 incorrectly enabled VM options ([#24924](https://projects.theforeman.org/issues/24924))
+
+#### Database
+* Deprecate SQLite production support ([#28248](https://projects.theforeman.org/issues/28248))
+
+#### JavaScript stack
+* eslint issue in the empty state component ([#28802](https://projects.theforeman.org/issues/28802))
+* main menu navigation to all entities doesn't work from the same entity page ([#28274](https://projects.theforeman.org/issues/28274))
+* Fix tests snapshots ([#28230](https://projects.theforeman.org/issues/28230))
+
+#### Packaging
```

Comment:
https://projects.theforeman.org/issues/28941 was just merged and I've updated it to set it fixed in 1.23.2.

Reply:
Thanks, I'll update this.